### PR TITLE
update doc about show analyze status

### PR DESCRIPTION
**Pull request by xuyifangreeneyes**
### What is changed, added or deleted? (Required)
Update doc about `show analyze status` and `information_schema.analyze_status`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/32215

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-analyze-status.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ summary: 了解 information_schema 表 `ANALYZE_STATUS`。
 
 `ANALYZE_STATUS` 表提供正在执行的收集统计信息的任务以及有限条历史任务记录。
 
+在 TiDB v6.1 之前，`ANALYZE_STATUS` 显示实例级别的任务，且 TiDB 重启后任务记录会被清空。从 TiDB v6.1 起，`ANALYZE_STATUS` 显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。
+
+从 TiDB v6.1 起，可以通过系统表 `mysql.analyze_jobs` 查看更早的（7 天内的） 历史记录。
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:12]:
```suggestion
从 TiDB v6.1 起，可以通过系统表 `mysql.analyze_jobs` 查看过去 7 天内的历史任务记录。
```

File: information-schema/information-schema-analyze-status.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ summary: 了解 information_schema 表 `ANALYZE_STATUS`。
 
 `ANALYZE_STATUS` 表提供正在执行的收集统计信息的任务以及有限条历史任务记录。
 
+在 TiDB v6.1 之前，`ANALYZE_STATUS` 显示实例级别的任务，且 TiDB 重启后任务记录会被清空。从 TiDB v6.1 起，`ANALYZE_STATUS` 显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:10]:
```suggestion
从 TiDB v6.1 起，`ANALYZE_STATUS` 表显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。在 TiDB v6.1 之前，`ANALYZE_STATUS` 仅显示实例级别的任务，且 TiDB 重启后任务记录会被清空。
```

File: information-schema/information-schema-analyze-status.md [hunk 3]
```diff
@@ -21,40 +25,50 @@ DESC analyze_status;
...
+8 rows in set (0.01 sec)
 ```
 
 `ANALYZE_STATUS` 表中列的含义如下：
 
 * `TABLE_SCHEMA`：表所属的数据库的名称。
 * `TABLE_NAME`：表的名称。
 * `PARTITION_NAME`：分区表的名称。
-* `JOB_INFO`：`ANALYZE` 任务的信息。
+* `JOB_INFO`：`ANALYZE` 任务的信息。如果分析索引则会包含索引名。`tidb_analyze_version =2` 情况下的任务则会包含采样率等配置项。
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:67]:
```suggestion
* `JOB_INFO`：`ANALYZE` 任务的信息。如果分析索引，该信息会包含索引名。当 `tidb_analyze_version =2` 时，该信息会包含采样率等配置项。
```

File: mysql-schema.md [hunk 4]
```diff
@@ -26,6 +26,7 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `stats_buckets` 统计信息的桶
 * `stats_histograms` 统计信息的直方图
 * `stats_meta` 表的元信息，比如总行数和修改数
+* `analyze_jobs` 正在执行的收集统计信息的任务以及 7 天内的历史任务记录。
```
Comment (qiancai) [inline mysql-schema.md:29]:
```suggestion
* `analyze_jobs` 正在执行的统计信息收集任务以及过去 7 天内的历史任务记录。
```

File: sql-statements/sql-statement-show-analyze-status.md [hunk 5]
```diff
@@ -8,40 +8,69 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-analyze-status/']
 
 `SHOW ANALYZE STATUS` 语句提供 TiDB 正在执行的统计信息收集任务以及有限条历史任务记录。
 
-## 语法图
+在 TiDB v6.1 之前，`SHOW ANALYZE STATUS` 显示实例级别的任务，且 TiDB 重启后任务记录会被清空。从 TiDB v6.1 起，`SHOW ANALYZE STATUS` 显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。
```
Comment (qiancai) [inline sql-statements/sql-statement-show-analyze-status.md:11]:
```suggestion
从 TiDB v6.1.0 起，执行 `SHOW ANALYZE STATUS` 语句将显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。在 TiDB v6.1.0 之前，执行 `SHOW ANALYZE STATUS` 语句仅显示实例级别的任务，且 TiDB 重启后任务记录会被清空。
```

File: sql-statements/sql-statement-show-analyze-status.md [hunk 6]
```diff
@@ -8,40 +8,69 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-analyze-status/']
...
 
-**ShowStmt:**
+从 TiDB v6.1 起，可以通过系统表 `mysql.analyze_jobs` 查看更早的（7 天内的） 历史记录。
```
Comment (qiancai) [inline sql-statements/sql-statement-show-analyze-status.md:13]:
```suggestion
从 TiDB v6.1.0 起，你可以通过系统表 `mysql.analyze_jobs` 查看过去 7 天内的历史记录。
```

File: statistics.md [hunk 7]
```diff
@@ -415,17 +415,25 @@ SHOW ANALYZE STATUS [ShowLikeOrWhere];
...
+目前 `SHOW ANALYZE STATUS` 会输出 11 列，具体如下：
 
 | 列名 | 说明            |
 | -------- | ------------- |
 | table_schema  |  数据库名    |
 | table_name | 表名 |
 | partition_name| 分区名 |
-| job_info | 任务具体信息。如果分析索引则会包含索引名 |
-| row_count | 已经分析的行数 |
+| job_info | 任务具体信息。如果分析索引则会包含索引名。`tidb_analyze_version =2` 情况下的任务则会包含采样率等配置项。 |
```
Comment (qiancai) [inline statistics.md:425]:
```suggestion
| job_info | 任务具体信息。如果分析索引，该信息会包含索引名。当 `tidb_analyze_version =2` 时，该信息会包含采样率等配置项。 |
```

File: statistics.md [hunk 8]
```diff
@@ -415,17 +415,25 @@ SHOW ANALYZE STATUS [ShowLikeOrWhere];
...
+| processed_rows | 已经分析的行数 |
 | start_time | 任务开始执行的时间 |
+| end_time | 任务结束执行的时间 |
 | state | 任务状态，包括 pending（等待）、running（正在执行）、finished（执行成功）和 failed（执行失败）|
+| fail_reason | 任务失败的原因。如果执行成功则为 `NULL`。 |
+| instance | 执行任务的 TiDB 实例 |
+| process_id | 执行任务的 process ID |
+
+在 TiDB v6.1 之前，`SHOW ANALYZE STATUS` 显示实例级别的任务，且 TiDB 重启后任务记录会被清空。从 TiDB v6.1 起，`SHOW ANALYZE STATUS` 显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。
```
Comment (qiancai) [inline statistics.md:434]:
```suggestion
从 TiDB v6.1.0 起，执行 `SHOW ANALYZE STATUS` 语句将显示集群级别的任务，且 TiDB 重启后仍能看到重启之前的任务记录。在 TiDB v6.1.0 之前，执行 `SHOW ANALYZE STATUS` 语句仅显示实例级别的任务，且 TiDB 重启后任务记录会被清空。
```

File: statistics.md [hunk 9]
```diff
@@ -415,17 +415,25 @@ SHOW ANALYZE STATUS [ShowLikeOrWhere];
...
+
+`SHOW ANALYZE STATUS` 仅显示最近的任务记录。从 TiDB v6.1 起，可以通过系统表 `mysql.analyze_jobs` 查看更早的（7 天内的） 历史记录。
```
Comment (qiancai) [inline statistics.md:436]:
```suggestion
`SHOW ANALYZE STATUS` 仅显示最近的任务记录。从 TiDB v6.1 起，你可以通过系统表 `mysql.analyze_jobs` 查看过去 7 天内的历史记录。
```

File: statistics.md [hunk 10]
```diff
@@ -330,29 +330,22 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
-    ```sql
-    KILL TIDB ${id};
-    ```
-
-    `${id}` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
-
-    > **注意：**
-    >
-    > 只有当使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例时，执行 `KILL` 语句才能终止后台的 `ANALYZE` 任务。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。更多信息，请参考 [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)。
+   - 如果 `enable-global-kill` 的值为 `true` (默认为 `true`）, 你可以直接执行 `KILL TIDB ID;` 语句。`ID` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
```
Comment (qiancai) [inline statistics.md:345]:
```suggestion
   - 如果 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 的值为 `true` (默认为 `true`）, 你可以直接执行 `KILL TIDB ID;` 语句。`ID` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
```

File: statistics.md [hunk 11]
```diff
@@ -330,29 +330,22 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
-    KILL TIDB ${id};
-    ```
-
-    `${id}` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
-
-    > **注意：**
-    >
-    > 只有当使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例时，执行 `KILL` 语句才能终止后台的 `ANALYZE` 任务。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。更多信息，请参考 [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)。
+   - 如果 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 的值为 `true` (默认为 `true`）, 你可以直接执行 `KILL TIDB ID;` 语句。`ID` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
+   - 如果 `enable-global-kill` 的值为 `false`, 你需要先使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例，然后再执行 `KILL TIDB ID;` 语句。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。
```
Comment (qiancai) [inline statistics.md:346]:
```suggestion
   - 如果 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 的值为 `true` (默认为 `true`），你可以直接执行 `KILL TIDB ID;` 语句。`ID` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
   - 如果 `enable-global-kill` 的值为 `false`，你需要先使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例，然后再执行 `KILL TIDB ID;` 语句。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。
```

File: statistics.md [hunk 12]
```diff
@@ -330,29 +330,22 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
-    KILL TIDB ${id};
-    ```
-
-    `${id}` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
-
-    > **注意：**
-    >
-    > 只有当使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例时，执行 `KILL` 语句才能终止后台的 `ANALYZE` 任务。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。更多信息，请参考 [`KILL [TIDB]`](/sql-statements/sql-statement-kill.md)。
+   - 如果 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 的值为 `true` (默认为 `true`），你可以直接执行 `KILL TIDB ID;` 语句。`ID` 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
+   - 如果 `enable-global-kill` 的值为 `false`，你需要先使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例，然后再执行 `KILL TIDB ID;` 语句。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。
```
Comment (qiancai) [inline statistics.md:346]:
```suggestion
   - 如果 [`enable-global-kill`](/tidb-configuration-file.md#enable-global-kill-从-v610-版本开始引入) 的值为 `true` (默认为 `true`），你可以直接执行 `KILL TIDB ${id};` 语句。其中，${id} 为上一步中查询得到的后台 `ANALYZE` 任务的 `ID`。
   - 如果 `enable-global-kill` 的值为 `false`，你需要先使用客户端连接到执行后台 `ANALYZE` 任务的 TiDB 实例，然后再执行 `KILL TIDB ${id};` 语句。如果使用客户端连接到其他 TiDB 实例，或者客户端和 TiDB 中间有代理，`KILL` 语句不能终止后台的 `ANALYZE` 任务。
```

File: sql-statements/sql-statement-show-analyze-status.md [hunk 13]
```diff
@@ -8,40 +8,69 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-analyze-status/']
...
 ## 示例
 
-{{< copyable "sql" >}}
-
 ```sql
-create table t(x int, index idx(x)) partition by hash(x) partition 4;
-analyze table t;
-show analyze status;
-```
+mysql> create table t(x int, index idx(x)) partition by hash(x) partitions 4;
```
[the remark this answers is not in the document]
Comment (xuyifangreeneyes) [reply sql-statements/sql-statement-show-analyze-status.md]: updated.

**Review by chrysan — APPROVED** [review]

File: information-schema/information-schema-analyze-status.md [hunk 14]
```diff
@@ -21,40 +25,48 @@ DESC analyze_status;
...
 +----------------+---------------------+------+------+---------+-------+
-7 rows in set (0.00 sec)
+11 rows in set (0.00 sec)
 ```
 
 {{< copyable "sql" >}}
 
 ```sql
-SELECT * FROM `ANALYZE_STATUS`;
+select * from information_schema.analyze_status;
```
Comment (shichun-0415) [inline information-schema/information-schema-analyze-status.md:43]:
```suggestion
SELECT * FROM information_schema.analyze_status;
```

File: mysql-schema.md [hunk 15]
```diff
@@ -26,6 +26,7 @@ aliases: ['/docs-cn/dev/system-tables/system-table-overview/','/docs-cn/dev/refe
 * `stats_buckets` 统计信息的桶
 * `stats_histograms` 统计信息的直方图
 * `stats_meta` 表的元信息，比如总行数和修改数
+* `analyze_jobs` 正在执行的统计信息收集任务以及过去 7 天内的历史任务记录。
```
Comment (shichun-0415) [inline mysql-schema.md:29]:
```suggestion
* `analyze_jobs` 正在执行的统计信息收集任务以及过去 7 天内的历史任务记录
```

**Review by shichun-0415 — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**